plugins/Hartree_Fock/SCF.irp.f

```diff
@@ -5,6 +5,7 @@ program scf
 ! output: hartree_fock.energy
 ! optional: mo_basis.mo_coef
   END_DOC
+!  call debug
```

Comment:
Maybe remove this line, or we can put a environement variable to control this kind one thing